packages/@o3r/application/schemas/cms.schema.json

```
@@ -31,9 +31,36 @@
       "type": "string",
       "description": "The relative path to the functional contents metadata folder to import"
     },
+    "agateEnabled": {
+      "type": "boolean",
+      "description": "Specify if Agate is enabled or not in the preview"
+    },
+    "baseNpmName": {
+      "type": "string",
+      "description": "if agateEnabled is set to true, this is used to set the proper Agate form type in the preview"
+    },
+    "features": {
+      "type": "object",
+      "description": "Contains features to be enabled or disabled",
+      "properties": {
+        "themingCssOverrideEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS override in theming"
+        },
+        "themingCssVariablesEnabled ": {
+          "type": "boolean",
+          "description": "Enable CSS variables in theming"
```

Maybe the following sentence would be more precised:
```suggestion
          "description": "Enable CSS variables customization in CMS theming administration"
```

(same for `themingCssOverrideEnabled`)